src/modules/service-catalog/components/useItemFormFields.spec.ts

```diff
@@ -0,0 +1,394 @@
+import { act, renderHook } from "@testing-library/react-hooks";
+import type { ServiceCatalogItem } from "../data-types/ServiceCatalogItem";
+import { useItemFormFields } from "./useItemFormFields";
+
+describe("useItemFormFields", () => {
+  const mockedItem: ServiceCatalogItem = {
```

Comment:
Nit: no need to call it `mocked*` naming in the specs. `item` or `serviceCatalogItem` would be simpler as we do in the following variables.